views/js/assets/strategies.js

```diff
@@ -54,6 +54,20 @@ define([
      */
     var strategies = {
 
+        externalReplaced : {
+            name : 'externalReplaced',
+            handle : function handlePackedUrl(url, data){
```

Comment:
make the function name reflect the strategy name. It's a convention only but make the  debugging much easier.